beacon-chain/core/transition/state-bellatrix.go

```diff
@@ -71,7 +70,7 @@ func GenesisBeaconStateBellatrix(ctx context.Context, deposits []*ethpb.Deposit,
 		return nil, err
 	}
 
-	st, err = b.ProcessPreGenesisDeposits(ctx, st, deposits)
+	st, err = altair.ProcessPreGenesisDeposits(ctx, st, deposits)
```

Comment:
I wonder if I should alias this?

Reply:
Let's leave this for now until we do a bigger code cleanup to move this into a bellatrix package ( right now one doesn't exist)